ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -102,8 +99,8 @@ async fn wait_for_content(
 
     let mut counter = 0;
     while received_content_value == HistoryContentItem::Unknown("".to_owned()) && counter < 5 {
-        error!("Retrying after 0.5sec, because content should have been present");
-        thread::sleep(time::Duration::from_millis(500));
+        error!("Retrying after 1s, because content should have been present");
+        thread::sleep(time::Duration::from_millis(1000));
```

Comment:
A code comment explaining the intuition of why not to drop it to 0.5s would be nice.

Reply:
I was able to move the retry duration back to 0.5 seconds with no failure.